doc/arkode/guide/source/Mathematics.rst

```diff
@@ -2074,3 +2074,51 @@ step attempt, or fails with the minimum step size, then the integration is halte
 and an error is returned. In this case the user may need to employ other
 strategies as discussed in :numref:`ARKODE.Usage.ARKStep.Tolerances` and
 :numref:`ARKODE.Usage.ERKStep.Tolerances` to satisfy the inequality constraints.
+
+.. _ARKODE.Mathematics.Relaxation:
+
+Relaxation Methods
+==================
+
+For cases where the problem :eq:`ARKODE_IVP` is conservative or dissipative with
+respect to a smooth convex function :math:`\xi(y(t))`, it is desirable to have
+the numerical method preserve these properties. That is
+:math:`\xi(y_n) = \xi(y_{n-1}) = \ldots = \xi(y_{0})` for conservative problems
+and :math:`\xi(y_n) \leq \xi(y_{n-1})` for dissipative problems. To this end,
+ARKODE supports relaxation methods
+:cite:p:`ketcheson2019relaxation, kang2022entropy, ranocha2020relaxation` to
+ensure dissipation or preservation of a global function.
+
+Relaxation methods compute a new solution
+
+.. math::
+   y_r = y_{n-1} + r d = r y_n + (1 - r) y_{n - 1}
+   :label: ARKODE_RELAX_SOL
+
+where :math:`d \equiv h_n \sum_{i=1}^{s}(b^E_i f^E_i + b^I_i f^I_i)` is the update
+direction and the relaxation factor, :math:`r`, is selected to ensure
+conservation or dissipation. Given a second order method with
```

Comment:
How `\xi` is defined is application dependent so I'm not sure what we can say generally that here.

Reply:
Then could we at least add some citations to that sentence so that readers know where to look for more information?  Perhaps something like "... problems and :math:`\xi(y_n) \leq \xi(y_{n-1})` for dissipative problems (see [ref] and [ref] for additional details)."

Reply:
I've added a sentence pointing users to the references in this section for examples of conservative or dissipative problems and the citations there in which provide more information in topics like entropy conservative or stable discretizations.